tests/integration/conftest.py

```diff
@@ -1086,15 +1093,17 @@ def __init__(self,
             junit_xml_file_path (str): A file path to a JUnitXML file.
             ci_environment (str): The CI environment used to produce the JUnitXML file.
                 (Valid values: 'asc', 'mk8s')
+            metadata (dict): Provide a dictionary of arbitrary metadata for this runner. Useful for when you want to
+                attach extra fixture data to the runner.
 
         Raises:
             RuntimeError: Invalid value provided for the 'ci_environment' argument.
         """
 
         if ci_environment == 'asc':
-            self._global_props = DEFAULT_ASC_GLOBAL_PROPERTIES
+            self._global_props = deepcopy(DEFAULT_ASC_GLOBAL_PROPERTIES)
         elif ci_environment == 'mk8s':
-            self._global_props = DEFAULT_MK8S_GLOBAL_PROPERTIES
+            self._global_props = deepcopy(DEFAULT_MK8S_GLOBAL_PROPERTIES)
```

Comment:
To safely allow testers the ability to manipulate the global props on a fixture this has to be deep copied. Otherwise changes to one fixture's global props will affect ALL fixtures.